code/game/objects/items/weapons/storage/backpack.dm

```diff
@@ -90,6 +90,16 @@
 	qdel(src)
 	return
 
+/obj/item/weapon/storage/backpack/holding/emp_act(severity)
+	if(prob(50+severity))
+		for(var/obj/O in src.contents)
+			if(istype(src.loc, /mob))
+				var/mob/user = src.loc
+				O.loc = user.loc
+				user.visible_message("<span class = 'danger'>The Bluespace portal malfunctions, spitting out its contents onto the floor!</span>")
```

Comment:
Yeah, I had like 6 BoH's on the ground though so figured it was just all of them dumping contents.